zooniverse/front-end-monorepo
# Subject Group Viewer - Feb 2020 update

## PR Overview

Package: `lib-classifier`
Project: Survos Survey adjacent

This PR attempts to fix two minor issues with the Subject Group Viewer.

- [x] Zoom range changed from 0.5-2x to 1-4x
  - Context: users mentioned it didn't make sense to be able to _zoom out_ on cells. Sounds legit.
- [x] The UI for focused and selected need to be re-thought.
  - Context: users have provided feedback indicating they're not sure if clicking on a selected cell also "unselects" it. (It does, but it may not be visually apparent.)
  - Analysis: this is a UI issue. The "you have focus on the cell" visuals (thick blue borders) can be confused with the "you have selected a cell" visuals (thin blue borders, and blue tint overlay), possibly more apparent on non-black-and-white photos.

### Testing

URL:
- https://local.zooniverse.org:8080/?env=staging&project=1900&workflow=3412 - original project using black-and-white cells. The blue overlay for "selected cells" is very visible here.
- (TBC) - other projects using photo (colour) images. The blue overlay is a lot LESS visible, which is an issue for determining if a cell is "selected"

Testing method:
- Open the Classifier on any of the links above. ☝️ 
- Click on any cell (we'll call this Cell 1) to **select** it.
- Cell 1 now has **focus** (THICK blue border) and is **selected** (blue overlay/tint)
- Click on another cell (Cell 2) to **select** it.
- Cell 2 now has **focus** (THICK blue border) and is **selected** (blue overlay)
- Cell 1 should now have _no focus_ but remain **selected** (thin blue border + blue overlay)
- Click on Cell 2 to to **unselect** it
- Cell 2 should now have **focus** (thick blue border) but is _not selected_ (no overlay)
- Compare Cell 1 and Cell 2 visually. Judge if they're visually distinct from one another.

### Status

WIP
